src/e2e/resources/data/InstructorSessionIndividualExtensionPageE2ETest.json

```diff
@@ -188,8 +219,8 @@
       "feedbackSessionName": "First Session",
       "courseId": "tm.e2e.ISesIe.CS2104",
       "feedbackQuestionId": "1",
-      "giver": "ISesIe.danny.tmms@gmail.tmt",
-      "recipient": "ISesIe.danny.tmms@gmail.tmt",
+      "giver": "danny.tmms@gmail.tmt",
+      "recipient": "danny.tmms@gmail.tmt",
```

Comment:
could I check why the emails were changed here?

Reply:
@cedricongjh 
Issue1: notifications tag has been removed on Mar 26th here is the commit https://github.com/TEAMMATES/teammates/pull/12707/commits/eba76eaaffeade1cfda4f2176019679f41c3d5d0

Reply:
@cedricongjh all issues has been addressed and build is stable please review it.

Reply:
@suneelval  Hey you shouldnt have to change the email to make it work maybe there is some underlying issue

Reply:
@mingyuanc changes has been reverted and build is table please review.